### PR TITLE
LINK-1698 | Additions to gdpr api user data fetch

### PR DESCRIPTION
**Pull request by nicobav**
Adds to gdpr-api/ GET data:
 - Organizations where user is admin or regular user
 - Events created by user 
 
 Refs LINK-1698

### CODE REVIEW

File: helevents/models.py [hunk 1]
```diff
@@ -228,11 +229,31 @@ class User(AbstractUser, UserModelPermissionMixin, SerializableMixin):
...
+        publishers = Organization.objects.filter(
+            Q(admin_users=self) | Q(regular_users=self)
+        )
+
+        return [
+            org.serialize()
+            for org in SerializablePublisher.objects.filter(
+                id__in=publishers.values_list("id", flat=True)
+            )
+        ]
```
Comment (danipran) [inline helevents/models.py:255]: Would this work as well? 🤔 
```suggestion
        return [
            org.serialize()
            for org in SerializablePublisher.objects.filter(
                Q(admin_users=self) | Q(regular_users=self)
            )
        ]
```

**Review by danipran — APPROVED** [review]
LGTM! Left one suggestion.

File: helevents/models.py (hunk 1 above)
Comment (nicobav) [reply helevents/models.py:255]: yes it would 🙃  
There might have been readability / self documenting reasoning but I don't see it anymore 😄

**Review by danipran — APPROVED** [review]
LGTM, left one comment which might need addressing

File: events/models.py [hunk 2]
```diff
@@ -783,9 +838,50 @@ class Meta:
...
+        {"name": "short_description"},
+        {"name": "start_time"},
+        {"name": "end_time"},
+        {"name": "images"},
+        {"name": "keywords"},
+        {
+            "name": "publisher",
+            "accessor": lambda x: (f"{x.id} - {x.name}") if x else "",
+        },
+        {"name": "user_organization"},
```
[the remark this answers is not in the document]
Comment (nicobav) [reply events/models.py:860]: nope! Nice catch 🎣 
I'll remove that 👍

**Pull request merged**